### PR TITLE
Add Hacktoberfest 2022 badge to profile

### LINKED ISSUE #643: Give yourself the Hacktoberfest2022 badge!
---

### Note - only PRs from existing Virtual Coffee members participating in the Virtual Coffee Hacktoberfest Initiative will be accepted.

---

### First step - complete #13 if you haven't already!

If you haven't completed #13 already, it is perfectly fine to do that _and_ this in the same Pull Request.

### Steps:

- Find your member file (it will be `members/members/yourgithubuser.ts` where `yourgithubuser` is your user name on github.
- Near the end of the file, there is a `badges` array:
  ```js
  badges: [],
  ```
- Add `'Hacktoberfest2022'` to the array
- Save, commit, and file a Pull Request!
- Bonus: while you're in there, take a look at the `flare` section for a little bit of fun 😁

Our [original issue for adding your member profile](https://github.com/Virtual-Coffee/virtualcoffee.io/issues/13) has a lot of details about previewing locally and interacting with GitHub - definitely check that out if you'd like to preview locally!

### PR DESCRIPTION
**Pull request by adiati98**
* Add flare
* Edit Hashnode's username

## Linked Issue

#643 

## Description

- Add Hacktoberfest 2022 badge to Ayu's member profile.
- Add flare
- Edit Hashnode's username.

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

**Comment by adiati98** [comment]
Hi @danieltott,
I have some questions out of curiosity.

I generated a Personal Access Token and followed all the steps in the [original issue](https://github.com/Virtual-Coffee/virtualcoffee.io/issues/13).
But I couldn't preview the member page locally.
It keeps kicking me out from `yarn dev` or `yarn start`.

So I made the changes blindly and hoping that I can preview it on Netlify deploy preview generated on GH after I created PR.
But I got this error messages instead.

Netlify Application Error

I've looked around the closed PRs from other members, and I can preview them (except for the images).

Do you have any thoughts why this happen?
TIA! 😊

File: members/members/adiati98.ts [hunk 1]
```diff
@@ -42,8 +43,8 @@ export const adiati98: MemberObject = {
 		// { type: 'youtube', channelId: 'yourChannelId' }, OR { type: 'youtube', customUrl: 'https://www.youtube.com/c/yourCustomUrl' },
 		{ type: 'polywork', username: 'adiatiayu' },
 		// { type: 'medium', username: 'yourUserName' },
-		{ type: 'hashnode', username: 'ayuadiati' },
+		{ type: 'hashnode', username: 'adiatiayu' },
 		// { type: 'website', url: 'https://virtualcoffee.io', title: 'Title of link' },
 	],
-	badges: [],
+	badges: [Hacktoberfest2022],
```
Comment (danieltott) [inline members/members/adiati98.ts:49]: Hey @adiati98! This needs to be a string (wrapped in `'`) - that should fix it up:

```suggestion
	badges: ['Hacktoberfest2022'],
```

**Review by danieltott — CHANGES_REQUESTED** [review]

File: members/members/adiati98.ts (hunk 1 above)
Comment (adiati98) [reply members/members/adiati98.ts]: Thanks, @danieltott!
I've pushed the changes just now 😄

**Review by danieltott — APPROVED** [review]
david s. pumkpins

**Pull request merged**